### PR TITLE
Upgrade ffigen to 6.1.2

### PR DESCRIPTION
**Pull request by WonyoungChoi**
Remove underscore '_' prefix from the name of structs and enums in generated bindings. 

for example, `_bundle_t` could not be used externally because it is a private member. But now it is possible. 
```dart
ffi.Pointer<_bundle_t> bundle_create() {
    return _bundle_create();
}
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.1.1
+* Remove '_' prefix from generated structs and enums.
```
Comment (swift-kim) [inline CHANGELOG.md:2]: There should be a blank line between the version name and the description by convention.

File: configs/4.0/ffigen.yaml [hunk 2]
```diff
@@ -108,3 +108,11 @@ dart-bool: true
 
 typedef-map:
   "size_t": "Int32"
+
+structs:
+  rename:
+    '_(.*)': '$1'
```
Comment (swift-kim) [inline configs/4.0/ffigen.yaml:114]: I think double quotes (") should be used by local convention of this file.

Also consider removing blank lines between top-level elements for consistency.

File: lib/src/bindings/6.5/generated_bindings.dart [hunk 3]
```diff
@@ -180899,17 +180899,17 @@ abstract class csr_cs_core_usage_e {
   static const int CSR_CS_CORE_USAGE_SINGLE = 3;
 }
 
-class __csr_cs_context_s extends ffi.Opaque {}
+class _csr_cs_context_s extends ffi.Opaque {}
 
-class __csr_cs_malware_s extends ffi.Opaque {}
+class _csr_cs_malware_s extends ffi.Opaque {}
```
Comment (swift-kim) [inline lib/src/bindings/6.5/generated_bindings.dart:180904]: Is this by design or a coincidence? If latter, you may check carefully.

File: configs/4.0/ffigen.yaml (hunk 2 above)
Comment (WonyoungChoi) [reply configs/4.0/ffigen.yaml]: Replace all double quotes to single quote. And keep the blank lines between top-level elements for readability, but I add some more blank lines for the consistency.

File: lib/src/bindings/6.5/generated_bindings.dart (hunk 3 above)
Comment (WonyoungChoi) [reply lib/src/bindings/6.5/generated_bindings.dart:180904]: I removed multiple underscores.

File: configs/5.5/ffigen.yaml [hunk 4]
```diff
@@ -3,135 +3,139 @@ preamble: |
...
-  - "-I./rootstraps/5.5/usr/lib/glib-2.0/include/"
+  - '-I./rootstraps/5.5/usr/include/glib-2.0/'
+  - '-I./rootstraps/5.5/usr/lib/glib-2.0/include/'
 
-dart-bool: true
+structs:
+  dependency-only: opaque
 
-typedef-map:
-  "size_t": "Int32"
```
Comment (swift-kim) [inline configs/5.5/ffigen.yaml:137]: Is this no longer necessary?

File: configs/5.5/ffigen.yaml [hunk 5]
```diff
@@ -3,135 +3,139 @@ preamble: |
...
 
   # include glib directories
-  - "-I./rootstraps/5.5/usr/include/glib-2.0/"
-  - "-I./rootstraps/5.5/usr/lib/glib-2.0/include/"
+  - '-I./rootstraps/5.5/usr/include/glib-2.0/'
+  - '-I./rootstraps/5.5/usr/lib/glib-2.0/include/'
 
-dart-bool: true
+structs:
+  dependency-only: opaque
```
Comment (swift-kim) [inline configs/5.5/ffigen.yaml:137]: Do you have an example of what this exactly does?

File: pubspec.yaml [hunk 6]
```diff
@@ -12,6 +12,6 @@ dependencies:
   yaml: ^3.1.0
 
 dev_dependencies:
-  ffigen: ^3.0.0
+  ffigen: ^6.1.2
```
Comment (swift-kim) [inline pubspec.yaml:15]: You should update the minimum SDK version (line 7) as well. The generated code requires Dart SDK 2.13.0 (or 2.17.0) or higher.

**Review by swift-kim — COMMENTED** [review]
Please check the README file and `extensions.dart` and update them if necessary. For example, `Pointer<Int8>` is now `Pointer<Char>` and `Pointer<Int32>` is now `Pointer<Int>`.

Please also check `dart analyze` warnings.

File: configs/5.5/ffigen.yaml (hunk 4 above)
Comment (WonyoungChoi) [reply configs/5.5/ffigen.yaml:137]: Yes, `size_t`, `pid_t`, `time_t` are mapped well automatically in ffigen 6.1.2.

File: configs/5.5/ffigen.yaml (hunk 5 above)
Comment (WonyoungChoi) [reply configs/5.5/ffigen.yaml:137]: This replace unused structure to ffi.Opaque like `Eina_List`. We can save few lines...

File: README.md [hunk 7]
```diff
@@ -28,42 +28,41 @@ import 'package:tizen_interop/4.0/tizen.dart';
...
 
 // Getting an integer value from the Native API.
-final memSize = using((Arena arena) {
-  Pointer<Int32> pMemSize = arena();
-  if (tizen.runtime_info_get_physical_memory_size(pMemSize) == 0) {
-    return pMemSize.value;
-  }
+final int preferenceValue = using((Arena arena) {
+  final Pointer<Int> pValue = arena();
+  int ret = tizen.preference_get_int(
```
Comment (swift-kim) [inline README.md:52]: `ret` can be `final`. You may either 

- Use `final` for `ret`.
- Omit `final` from all other values.
- Use `if (..)`.

**Review by swift-kim — APPROVED** [review]

**Pull request merged**